modules/apps/commerce/commerce-service/service.xml

```diff
@@ -921,6 +921,7 @@
 		<exception>CommerceAddressType</exception>
 		<exception>CommerceAddressZip</exception>
 		<exception>CommerceGeocoder</exception>
+		<exception>CommerceInvalidOrderQuantity</exception>
```

Comment:
Use `CommerceOrderQuantity` as exception name. Resend me this PR with the updated change. Thanks

Reply:
should be good now